### DISCUSSION TITLE
Extract date from timeseries in to separate dataset

### DISCUSSION THREAD

**Discussion by stoormgeo** [insarlab/MintPy#863] [Q&A]
Hi,
Is there an easy way to extract an individual date from timeseries.h5 in to separate dataset?
I tried using the --dset parameter in geocode.py but it did not work.

**Comment by yunjunz** [comment]
You may want to try `save_gdal.py`.

**Reply by stoormgeo** [reply]
It works, but I need extract to .h5 format for processing by day with asc_desc2horz_vert.py.

**Reply by yunjunz** [reply]
`asc_desc2horz_vert.py` does not require HDF5 file, as far as I remember.
